web/robot/list_reordering.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import math
+import random
+
+class Point():
+    def __init__(self, x, y):
+        self.x = x;
+        self.y = y;
+
+    def __eq__(self, other):
+        return (self.x == other.x) & (self.y == other.y)
+
+
+def minDistance(start, remaininglist):
+
+        # Initilaize minimum distance for next node
+    min = sys.maxsize
+
+        # Search not nearest vertex not in the
+        # shortest path tree
+    for v in remaininglist:
+        dist = euclideanDistance(start, v)
+        if dist < min:
+            min = dist
+            min_point = v
+
+    return min_point
+
+def euclideanDistance(start, end):
+    return math.sqrt((start[0] - end[0])**2 +(start[1] - end[1])**2)
+
+
+
+    # Funtion that implements Dijkstra's single source
```

Comment:
Are these comments accurate for this version of the code?

Reply:
No, had left them in, will update soon.